ebpfsvc/rpc_interface.idl

```diff
@@ -13,59 +13,61 @@ import "wtypes.idl";
 // Interface Attributes
 //
 
-[
-    uuid(6bef171d-7205-4b63-a1e5-d00f01e6a0c1),
-    version(1.0),
-    pointer_default(unique)
-]
+[uuid(6bef171d-7205-4b63-a1e5-d00f01e6a0c1), version(1.0), pointer_default(unique)]
 
-interface ebpf_service_interface
-{
-    // eBPF instruction is 64-bit.
-    typedef uint64_t ebpf_instruction;
+    interface ebpf_service_interface {
 
-    typedef [system_handle(sh_file)] HANDLE file_handle_t;
+        // TODO: This is a duplicate of EbpfMapDescriptor structure
```

Comment:
Can you do this TODO?

Reply:
Actually EbpfMapDescriptor is defined in spec_type_descriptors.hpp and that header file includes <vector> and <string>.
So it does not look trivial to include spec_type_descriptors.hpp in the IDL file.

Reply:
Ok, understood.  Can you file an issue and put the issue # in the TODO comment then?